src/javascripts/ng-admin/es6/lib/DataStore/DataStore.js

```diff
@@ -0,0 +1,119 @@
+import Entry from "../Entry";
+
+class DataStore {
+    constructor() {
+        this._entries = {};
```

Comment:
May be a good occasion to use [ES6 Sets](http://www.nczonline.net/blog/2012/09/25/ecmascript-6-collections-part-1-sets/)?